addons/l10n_ir/data/account_fiscal_position_data.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+        <record
```

Comment:
usually account.fiscal.position.template and account.fiscal position.tax.template are put in two different file but it's a detail (ex: https://github.com/odoo/odoo/blob/15.0/addons/l10n_be/data/account_fiscal_position_tax_template_data.xml) 

Also there is a minor indentation problem :smile: